projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss

```diff
@@ -1,189 +1,148 @@
-    );
-    padding: $cx-navigation-child-item-trbl-padding;
+  &-child-item {
+    border-style: var(--cx-border-style, solid);
+    border-width: var(--cx-border-width, 0 0 1px);
+    border-color: var(--cx-border-color, var(--cx-g-color-background));
+    padding: var(--cx-padding, 0.45rem 1rem);
     &:hover {
-      @include var-color(
-        'background-color',
-        $cx-navigation-child-item-hover-background-color
+      background-color: var(
+        --cx-background-color,
+        var(--cx-g-color-transparent)
       );
     }
     &:last-child {
-      border-bottom: $cx-navigation-child-item-last-child-border;
+      border-style: var(--cx-border-style, none);
+      border-width: var(--cx-border-width, 0);
     }
     @include media-breakpoint-down(md) {
-      border-bottom: $cx-navigation-child-item-md-border;
+      border-style: var(--cx-border-style, none);
+      border-width: var(--cx-border-width, 0);
     }
   }
-  &__child-link {
-    @include var-color('color', $cx-navigation-child-link-color);
+  &-child-link {
     @include type('7');
+    color: var(--cx-color, var(--cx-g-color-text));
     &:hover {
-      @include var-color('color', $cx-navigation-child-link-hover-color);
-      text-decoration: $cx-navigation-child-link-hover-text-decoration;
+      color: var(--cx-color, var(--cx-g-color-primary));
+      text-decoration: var(--cx-text-decoration, none);
     }
   }
-  &__child-list-columns {
-    border-radius: $cx-navigation-child-list-columns-border-radius;
-    padding: $cx-navigation-child-list-columns-trbl-padding;
+  &-child-list-columns {
+    border-radius: var(--cx-border-radius, 0);
+    padding: var(--cx-padding, 0.75rem);
     &.show {
-      display: flex;
+      display: var(--cx-display, flex);
     }
     @include media-breakpoint-down(md) {
-      flex-direction: column;
-      position: static;
-      width: 100%;
-      border: $cx-navigation-child-list-columns-md-border;
-      margin: $cx-navigation-child-list-columns-md-trbl-margin;
+      flex-direction: var(--cx-flex-direction, column);
+      position: var(--cx-position, static);
+      width: var(--cx-width, 100%);
+      border-style: var(--cx-border-style, none);
+      border-width: var(--cx-border-width, 0);
+      margin: var(--cx-margin, 0);
     }
   }
   //column option
-  &__column-title {
-    text-transform: $cx-navigation-column-title-text-case;
-    @include type('7', $cx-navigation-column-title-font-weight);
-    display: block;
-    width: 100%;
-    padding: $cx-navigation-column-title-trbl-padding;
+  &-column-title {
+    @include type('7');
+    text-transform: var(--cx-text-transform, capitalize);
+    font-weight: var(--cx-font-weight, var(--cx-g-font-weight-bold));
+    display: var(--cx-display, block);
+    width: var(--cx-width, 100%);
+    padding: var(--cx-padding, 0.25rem 1.5rem);
   }
-  &__child-column {
-    margin-right: $cx-navigation-child-column-trbl-margin;
```

Comment:
Funny story (not an issue) but this rule didn't work before but now it works and it changes the look of the dropdown a bit 😉

Reply:
fixed!